tools/src/versioning/ios/index.ts

```diff
@@ -452,97 +354,18 @@ async function generateVersionedExpoAsync(
-
-  // some pods are optional, so those specs should be omitted from versioned code
-  const excludedPodNames = getExcludedPodNames();
-  const packages = (await getListOfPackagesAsync()).filter((pkg) => {
-    const podName = pkg.podspecName;
-    return podName && pkg.isVersionableOnPlatform('ios') && !excludedPodNames.includes(podName);
-  });
-
-  for (const pkg of packages) {
-    const modulePath = path.join(EXPO_DIR, RELATIVE_UNIVERSAL_MODULES_PATH, pkg.packageName);
-    const podName = pkg.podspecName!;
-
-    await fs.copy(path.join(modulePath, 'ios'), path.join(versionedExpoPath, podName));
-
-    // We're moving away from additional and unnecessary subdirectory.
-    // The source code may not be wrapped by the directory with pod's name (see ExpoModulesCore).
-    // So, move this dir only when it exists.
-    const versionedSourcesPath = path.join(versionedExpoPath, podName, podName);
-    if (await fs.pathExists(versionedSourcesPath)) {
-      await fs.move(
-        versionedSourcesPath,
-        path.join(versionedExpoPath, podName, versionedUnimodulePods[podName])
-      );
-    }
-
-    await fs.copy(
-      path.join(modulePath, 'package.json'),
-      path.join(versionedExpoPath, podName, 'package.json')
-    );
-  }
-
-  for (const originalUnimodulePodName of originalUnimodulePodNames) {
-    const versionedUnimodulePodName = versionedUnimodulePods[originalUnimodulePodName];
-
-    const originalPodSpecPath = path.join(
-      versionedExpoPath,
-      originalUnimodulePodName,
-      `${originalUnimodulePodName}.podspec`
-    );
-    const prefixedPodSpecPath = path.join(
-      versionedExpoPath,
-      originalUnimodulePodName,
-      `${versionedUnimodulePodName}.podspec`
-    );
-
-    if (!(await fs.pathExists(originalPodSpecPath))) {
-      continue;
-    }
-
-    console.log(`Generating podspec for ${chalk.green(originalUnimodulePodName)} ...`);
-
-    await fs.move(originalPodSpecPath, prefixedPodSpecPath);
-
-    // Replaces versioned modules in the podspec eg. 'EXCore' => 'ABI28_0_0EXCore'
-    // `E` flag is required for extended syntax which allows to use `(a|b)`
-    await spawnAsync('sed', [
-      '-Ei',
-      '--',
-      `s/'(${depsToReplace})('|\\/)/'${versionName}\\1\\2/g`,
-      prefixedPodSpecPath,
-    ]);
-    await spawnAsync('sed', [
-      '-i',
-      '--',
-      `s/React/${versionedReactPodName}/g`,
-      prefixedPodSpecPath,
-    ]);
-    await spawnAsync('sed', [
-      '-i',
-      '--',
-      `s/${versionName}UM${versionedReactPodName}/${versionName}UMReact/g`,
-      prefixedPodSpecPath,
-    ]);
-    await spawnAsync('sed', [
-      '-i',
-      '--',
-      "s/'..', 'package.json'/'package.json'/g",
-      prefixedPodSpecPath,
-    ]);
-  }
-
   console.log(`Generating podspec for ${chalk.green('ExpoKit')} ...`);
 
   await generateExpoKitPodspecAsync(
     versionedExpoKitPath,
     versionedUnimodulePods,
     versionName,
-    versionNumber
+    `${sdkNumber}.0.0`
```

Comment:
You've created some helper functions doing similar things 🤔 
Don't you want to add another doing this? Or is it the only place where it's needed? 🤔

Reply:
The helper you talk about is for `ABIx_0_0`, not `x.0.0`. Right now it's the only place where it's needed and I hope this part of code will be removed pretty soon either way 😄